security/access_control.rst

```diff
@@ -37,10 +37,10 @@ Take the following ``access_control`` entries as an example:
         security:
             # ...
             access_control:
-                - { path: ^/admin, roles: ROLE_USER_IP, ip: 127.0.0.1 }
-                - { path: ^/admin, roles: ROLE_USER_HOST, host: symfony\.com$ }
-                - { path: ^/admin, roles: ROLE_USER_METHOD, methods: [POST, PUT] }
-                - { path: ^/admin, roles: ROLE_USER }
+                - { path: '^/admin', roles: ROLE_USER_IP, ip: 127.0.0.1 }
+                - { path: '^/admin', roles: ROLE_USER_HOST, host: symfony\.com$ }
+                - { path: '^/admin', roles: ROLE_USER_METHOD, methods: [POST, PUT] }
+                - { path: '^/admin', roles: [ROLE_MANAGER, ROLE_ADMIN] }
```

Comment:
I've always disliked the array roles, because it's not obvious if the logic is OR or AND. I believe it's OR (?). That needs to be documented, probably with a comment above this line

Reply:
Yeah I would prefer people not to be confused about this, so I recommend we keep being explicit everywhere it's not obvious. Comment added, thanks!